src/main/webapp/dev/js/main/feedbackSubmissionsEdit.es6

```diff
@@ -963,10 +963,12 @@ $(document).ready(() => {
     if (typeof richTextEditorBuilder !== 'undefined') {
         $.each(textFields, (i, textField) => {
             const id = $(textField).attr('id');
+            const isSessionClosed = $(textField).data('is-session-closed') || false;
```

Comment:
I believe jQuery converts the case automatically:
```javascript
const isSessionClosed = $(textField).data('isSessionClosed') || false;
```
If we are going with my above suggestion, the fact that we're giving the benefit of the doubt that the session is still open becomes clearer:
```javascript
const isSessionOpen = $(textField).data('isSessionOpen') || true;
```